js/buda.js

```diff
@@ -262,9 +290,11 @@ module.exports = class buda extends Exchange {
                 'expiryDatetime': undefined,
                 'strike': undefined,
                 'optionType': undefined,
+                'taker': Precise.stringDiv (taker_fee, '1000'),
```

Comment:
We must return a safe number here – let's use `this.parseNumber (Precise.stringDiv (taker_fee, '1000'))` cause `Precise.stringDiv` returns a string.

Reply:
uh-oh, how i forgot it👍  (btw, you can just comment one word i.e. `parseNumber` under that highlighted line, and I will understand your comment, thus, you will save around 10+ seconds for such comments).